src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+    private int inedex = random.nextInt(Color.values().length);
+
+    public String getRandomColor() {
+        return Color.values()[inedex].name();
+    }
```

Comment:
with current approach your index will remain unchanged but you need it to change for every time method is called
```suggestion
public class ColorSupplier {
    private Random random = new Random();

    public String getRandomColor() {
        int index = random.nextInt(Color.values().length);
        return Color.values()[index].name();
    }
```

Reply:
I have corrected. but my solution also gives random color

Reply:
it gives random color for each time you start the program, not for each  time the method is called